### PR TITLE
Translate `ReflectionFiber` methods

### PR DESCRIPTION
**Pull request by Fan2Shrek**
Here is the translation of missing `ReflectionFiber` methods

- `ReflectionFiber::__construct()`
- `ReflectionFiber::getCallable()`
- `ReflectionFiber::getExecutingFile()`
- `ReflectionFiber::getExecutingLine()`
- `ReflectionFiber::getFiber()`
- `ReflectionFiber::getTrace()`

### CODE REVIEW

File: reference/reflection/reflectionfiber/getexecutingfile.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="ReflectionFiber">
+   <modifier>public</modifier> <type class="union"><type>string</type><type>null</type></type><methodname>ReflectionFiber::getExecutingFile</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
+   Renvoie le chemin complet et le nom du fichier du point d'éxécution actuel de la <classname>Fiber</classname> réfléchie.
```
Comment (devnexen) [inline reference/reflection/reflectionfiber/getexecutingfile.xml:17]: to be consisten with your own style above I would use `reflétée` instead.

File: reference/reflection/reflectionfiber/getexecutingfile.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Le chemin complet et le nom du fichier de la fibre réfléchie.
```
Comment (devnexen) [inline reference/reflection/reflectionfiber/getexecutingfile.xml:30]: ditto

File: reference/reflection/reflectionfiber/getexecutingline.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="ReflectionFiber">
+   <modifier>public</modifier> <type class="union"><type>int</type><type>null</type></type><methodname>ReflectionFiber::getExecutingLine</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
+   Renvoie le numéro de ligne du point d'éxécution actuel de la <classname>Fiber</classname> réfléchie.
```
Comment (devnexen) [inline reference/reflection/reflectionfiber/getexecutingline.xml:17]: ditto

File: reference/reflection/reflectionfiber/getfiber.xml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="ReflectionFiber">
+   <modifier>public</modifier> <type>Fiber</type><methodname>ReflectionFiber::getFiber</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
+   Renvoie l'instance de la <classname>Fiber</classname> réfléchie.
```
Comment (devnexen) [inline reference/reflection/reflectionfiber/getfiber.xml:17]: ditto

File: reference/reflection/reflectionfiber/getfiber.xml [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   L'instance de la <classname>Fiber</classname> réfléchie.
```
Comment (devnexen) [inline reference/reflection/reflectionfiber/getfiber.xml:29]: ditto

File: reference/reflection/reflectionfiber/gettrace.xml [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="ReflectionFiber">
+   <modifier>public</modifier> <type>array</type><methodname>ReflectionFiber::getTrace</methodname>
+   <methodparam choice="opt"><type>int</type><parameter>options</parameter><initializer><constant>DEBUG_BACKTRACE_PROVIDE_OBJECT</constant></initializer></methodparam>
+  </methodsynopsis>
+  <para>
+   Renvoie la trace d'appels du point d'éxécution actuel de la <classname>Fiber</classname> réfléchie.
```
Comment (devnexen) [inline reference/reflection/reflectionfiber/gettrace.xml:17]: ditto

File: reference/reflection/reflectionfiber/getexecutingfile.xml [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Le chemin complet et le nom du fichier de la fibre reflétée.
+   Si la fibre réfléchie est utilisée en dehors d'une fonction définie par l'utilisateur, &null; est renvoyé.
```
Comment (devnexen) [inline reference/reflection/reflectionfiber/getexecutingfile.xml:31]: Sorry just notice one instance here.

File: reference/reflection/reflectionfiber/getfiber.xml [hunk 8]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- EN-Revision: ec2fe9a592f794978114ef5021db9f1d00c2e05d Maintainer: Fan2Shrek Status: ready -->
+<!-- Reviewed: yes -->
+<refentry xml:id="reflectionfiber.getfiber" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>ReflectionFiber::getFiber</refname>
+  <refpurpose>Renvoie l'instance de la Fibre réfléchie</refpurpose>
```
Comment (devnexen) [inline reference/reflection/reflectionfiber/getfiber.xml:7]: ditto

File: reference/reflection/reflectionfiber/getexecutingline.xml [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="ReflectionFiber">
+   <modifier>public</modifier> <type class="union"><type>int</type><type>null</type></type><methodname>ReflectionFiber::getExecutingLine</methodname>
+   <void/>
+  </methodsynopsis>
+  <para>
+   Renvoie le numéro de ligne du point d'éxécution actuel de la <classname>Fiber</classname> reflétée.
+   Si la fibre réfléchie est utilisée en dehors d'une fonction définie par l'utilisateur, &null; est renvoyé.
```
Comment (devnexen) [inline reference/reflection/reflectionfiber/getexecutingline.xml:18]: ditto

File: reference/reflection/reflectionfiber/getexecutingfile.xml (hunk 7 above)
Comment (Fan2Shrek) [reply reference/reflection/reflectionfiber/getexecutingfile.xml]: Np, my bad too :p 

Should be done now :)

**Review by devnexen — APPROVED** [review]

**Pull request merged**